Sources/SwiftSyntaxBuilder/gyb_generated/DeclBuildables.swift

```diff
@@ -0,0 +1,1082 @@
+    inheritanceClause: TypeInheritanceClause? = nil,
+    genericWhereClause: GenericWhereClause? = nil,
+    members: MemberDeclBlock
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      extensionKeyword: extensionKeyword,
+      extendedType: extendedType,
+      inheritanceClause: inheritanceClause,
+      genericWhereClause: genericWhereClause,
+      members: members
+    )
+  }
+}
+
+extension MemberDeclBlock {
+  public init(
+    leftBrace: TokenSyntax = Tokens.`leftBrace`,
+    @MemberDeclListBuilder membersBuilder: () -> MemberDeclList = { .empty },
+    rightBrace: TokenSyntax = Tokens.`rightBrace`
+  ) {
+    self.init(
+      leftBrace: leftBrace,
+      members: membersBuilder(),
+      rightBrace: rightBrace
+    )
+  }
+}
+
+extension SourceFile {
+  public init(
+    @CodeBlockItemListBuilder statementsBuilder: () -> CodeBlockItemList = { .empty },
+    eofToken: TokenSyntax
+  ) {
+    self.init(
+      statements: statementsBuilder(),
+      eofToken: eofToken
+    )
+  }
+}
+
+extension FunctionParameter {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    firstName: TokenSyntax? = nil,
+    secondName: TokenSyntax? = nil,
+    colon: TokenSyntax? = Tokens.`colon`,
+    type: TypeBuildable? = nil,
+    ellipsis: TokenSyntax? = Tokens.`ellipsis`,
+    defaultArgument: InitializerClause? = nil,
+    trailingComma: TokenSyntax? = Tokens.`comma`
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      firstName: firstName,
+      secondName: secondName,
+      colon: colon,
+      type: type,
+      ellipsis: ellipsis,
+      defaultArgument: defaultArgument,
+      trailingComma: trailingComma
+    )
+  }
+}
+
+extension FunctionDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    funcKeyword: TokenSyntax = Tokens.`func`,
+    identifier: TokenSyntax,
+    genericParameterClause: GenericParameterClause? = nil,
+    signature: FunctionSignature,
+    genericWhereClause: GenericWhereClause? = nil,
+    body: CodeBlock? = nil
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      funcKeyword: funcKeyword,
+      identifier: identifier,
+      genericParameterClause: genericParameterClause,
+      signature: signature,
+      genericWhereClause: genericWhereClause,
+      body: body
+    )
+  }
+}
+
+extension InitializerDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    initKeyword: TokenSyntax = Tokens.`init`,
+    optionalMark: TokenSyntax? = nil,
+    genericParameterClause: GenericParameterClause? = nil,
+    parameters: ParameterClause,
+    throwsOrRethrowsKeyword: TokenSyntax? = nil,
```

Comment:
Keywords with an `or` in do we ignore

Reply:
I’m not quite sure what you’re trying to say, but I don’t think there’s much we can do here. The user needs to choose between three options (no keyword, `throws` or `rethrows`).